cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,67 @@
+import ast
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
+            '--markets',
+            nargs='?',
+            help='String representation of a list of markets'
+        )
+
+    def get_data_snapshots(self):
+        """ Gets all data snapshots from browse pages
+        Assumes there is one data snapshot per page
+        """
+        snapshots = []
+        for page in BrowsePage.objects.all():
```

Comment:
@chosak   I totally understand your concern here -- I have a few thoughts.
- I didn't seriously consider using a Django model for this because this is something that feels ripe for a CMS. While I would love for Wagtail to support programmatic updates of fields, I think that's a trade-off we've made (which doesn't feel relevant to this specific PR).
- Snippets is an interesting idea but it would deviate from how we're currently using them (for things that are "reusable" somehow).  I think it'd add a layer of abstraction for the editors trying to find and update text fields on a data snapshot since it's somewhat hidden from view on a page itself.  `ReusableText` is the most recent snippet and we had a lot of conversations around its usage.  I know you're more familiar with the Job Listings and the reasoning behind modeling it in Django, but I would be wary of making a habit of building things for our CMS outside of the CMS.
- I think it's OK that we republish this page, don't see that as a con. 

So, that all said, I agree it's not ideal to have to update data this way.  It is brittle, but if the script were to ever stop working, our tests (as I mentioned I plan to add as part of this PR) would fail.  Additionally, this script that will probably get run only once a month.